src/newItinerary/Place.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+
+import { Bullet, BulletTypes } from '../bullet'
+import { Proximity } from '../proximity'
+import { ItineraryItem, ItineraryItemProps } from './internals/ItineraryItem'
+import {
+  StyledChevronIcon,
+  StyledContent,
+  StyledLabel,
+  StyledProximity,
+  StyledSubLabel,
+} from './Place.style'
```

Comment:
I know, naming things is pretty hard.. but why not associate this with your components following a compound pattern? eg: `Place.Label`

Reply:
I'll let you add a topic in the chapter. I just reused the naming "convention" we have everywhere else.